scripts/apps/monitoring/views/monitoring-view.html

```diff
@@ -5,15 +5,31 @@
                 <div class="monitoring__toolbar-container">
                     <div class="subnav" ng-if="hideMonitoringToolbar1 !== true">
                         <div class="flat-searchbar monitoring-backlink" ng-if="monitoring.singleGroup">
-                            <button sd-tooltip="{{ :: 'Back to original view' | translate }}" flow="right" class="navbtn navbtn--left" ng-click="monitoring.viewMonitoringHome()">
+                            <button sd-tooltip="{{ gettext('Back to original view') }}" flow="right" class="navbtn navbtn--left" ng-click="monitoring.viewMonitoringHome()">
                                 <i class="icon-arrow-left"></i>
                             </button>
                         </div>
-                                        
+
+                        <!-- Filter content using content profile -->
```

Comment:
can you make it [scrollable](https://user-images.githubusercontent.com/2076953/72523216-a173f080-385f-11ea-9e46-c5d0dfe9e2b2.png)